.rubocop_todo.yml

```diff
@@ -170,7 +176,6 @@ Metrics/BlockLength:
     - 'spec/factories/subscription_factory.rb'
     - 'spec/factories/user_factory.rb'
     - 'spec/factories/variant_factory.rb'
-    - 'spec/requests/api/orders_spec.rb'
```

Comment:
After cleaning local files and rebasing the branch, I see I'm still removing these files...
I think this is not expected, as I don't really touch them in previous commits.

Maybe this is because the file was not updated for a while now? I'm not to sure.

Anyway, for this particular case, indeed running 'rubocop spec/requests/api/orders_spec.rb' returns no offenses, so this seems correct.